configure

```diff
@@ -49,6 +49,76 @@ while true; do
   # Retry
 done
 
+## Set up MKL related environment settings
+
+while [ "$TF_NEED_MKL" == "" ]; do
+	fromuser=""
+  read -p "Do you wish to build TensorFlow with MKL support? [y/N] " INPUT
```

Comment:
right, is it ok to put this in "if false" block for now, or should I comment this code?